### PR TITLE
17 key Panasonic rotary encoder BLE pad

### PR DESCRIPTION
**Pull request by haierwangwei2005**
## Types of Changes

- [x] Core
- [x] Bugfix
- [x] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by tzarc** [comment]
Please limit your changes to the board you want to merge.

**Comment by haierwangwei2005** [comment]
> Please limit your changes to the board you want to merge.

Thank you sir,I have delete the redundant one.

File: keyboards/latinpadble/config.h [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+#pragma once
+
+#include "config_common.h"
+#define VENDOR_ID       0x7C88 // "hw" = haierwangwei2005
+#define PRODUCT_ID      0x7C98 // "lp" = latinpadble
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    haierwangwei2005
+#define PRODUCT         latinpadble
+#define DESCRIPTION     Panasonic rotary encoder BLE pad
```
Comment (drashna) [inline keyboards/latinpadble/config.h:25]:
```suggestion
```

File: keyboards/latinpadble/config.h [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+#define NO_ACTION_LAYER
+#define NO_ACTION_TAPPING
+#define NO_ACTION_ONESHOT
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
```
Comment (drashna) [inline keyboards/latinpadble/config.h:32]: Do you need all of these disabled?  If not, you're losing out on a lot of functionality. 

```suggestion
// #define NO_ACTION_LAYER
// #define NO_ACTION_TAPPING
// #define NO_ACTION_ONESHOT
#define NO_ACTION_MACRO
#define NO_ACTION_FUNCTION

```

File: keyboards/latinpadble/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/latinpadble/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/latinpadble/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_pad( 
+        KC_PGUP,
+	    KC_KP_7, KC_KP_8, KC_KP_9, MO(1),
+        KC_P4, KC_P5, KC_P6, KC_KP_PLUS,
+        KC_P1, KC_P2, KC_P3, KC_KP_MINUS,
+        KC_P0, KC_PDOT,KC_DELETE, KC_KP_ENTER),
+    [1] = LAYOUT_pad(
```
Comment (drashna) [inline keyboards/latinpadble/keymaps/default/keymap.c:9]: This won't work with `NO_ACTION_LAYER`

File: keyboards/latinpadble/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,66 @@
+ /* Keymap _0: (Base Layer) Default Layer
```
Comment (drashna) [inline keyboards/latinpadble/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/latinpadble/lib/glcdfont.c [hunk 6]
```diff
@@ -0,0 +1,228 @@
+#include "progmem.h"
```
Comment (drashna) [inline keyboards/latinpadble/lib/glcdfont.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/latinpadble/keymaps/default/keymap.c (hunk 4 above)
Comment (haierwangwei2005) [reply keyboards/latinpadble/keymaps/default/keymap.c:24]: Thanks a lot sir

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Comment by latincompass** [comment]
Thanks a lot ,sir

**Pull request merged**